recipes/xgboost/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "xgboost" %}
+{% set version = "0.6a2" %}
```

Comment:
So normally we try to only do formal releases. However, if there isn't one, I don't want to be pedantic. Just once there is a formal release, let's package it. If you want to do more dev releases after the formal release, ping me and we can talk.